homeassistant/components/wiz_light/config_flow.py

```diff
@@ -0,0 +1,62 @@
+"""Config flow for wiz_light."""
+import logging
+import re
+
+from pywizlight import wizlight
+from pywizlight.exceptions import WizLightConnectionError, WizLightTimeOutError
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_IP_ADDRESS, CONF_NAME
+from homeassistant.data_entry_flow import AbortFlow
+
+from .const import DEFAULT_NAME, DOMAIN  # pylint: disable=unused-import
```

Comment:
Both of these imports are used. Why does pylint need to be disabled?

Reply:
I don't know why this pups up in the test stack. In my devcontainer pylint doesn't shows me an error.